lib/still/preprocessor/profiler.ex

```diff
@@ -19,11 +19,9 @@ defmodule Still.Preprocessor.Profiler do
   end
 
   @impl true
-  def after_render(%{metadata: %{start_time: start_time}} = file) do
-    if should_profile?(file) do
-      end_time = Profiler.timestamp()
-      Profiler.register(file, end_time - start_time)
-    end
+  def after_render(%{metadata: %{_profiler_start_time: start_time}} = file) do
```

Comment:
Isn't it a convention to use `__var_name__` in Elixir? Looking at `__struct__` and what Ecto schemas add

Reply:
That's a convention, but I'm not sure what it means. What is it? Like "private" variables?

Reply:
I think so. Private or restricted struct keys.

Reply:
👍